x-pack/test/api_integration/apis/synthetics/add_monitor_private_location.ts

```diff
@@ -376,7 +376,7 @@ export default function ({ getService }: FtrProviderContext) {
           (pkgPolicy: PackagePolicy) =>
             pkgPolicy.id === monitorId + '-' + testFleetPolicyID + `-default`
         );
-        expect(packagePolicyAfterUpgrade.package.version).eql('0.11.2');
+        expect(packagePolicyAfterUpgrade.package.version).eql('0.11.3');
```

Comment:
@elastic/uptime I had to update this test due to the `synthetics` package has been updated in `fleet_packages.json` file for version 8.6.

I was wondering if this assertion in this test could be updated (or removed?) to not fail in these PRs that update the bundled packages.

Reply:
@dominiqueclarke - Could you take a look at this? Would love to get an answer here so we can move forward with automating the merge of these PR's in the future. Thanks!

Reply:
That's fine - I'll merge and follow up with the Synthetics folks when they're online.